Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Kiosk](https://hosted.weblate.org/projects/kimai/kiosk/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/kimai/kimai?pullRequest=5318) <br/>All committers have signed the CLA.

weblate committed:
Translated using Weblate (Hungarian)

Currently translated at 66.5% (323 of 485 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (Turkish)

Currently translated at 99.5% (483 of 485 strings)

Translated using Weblate (Czech)

Currently translated at 50.0% (1 of 2 strings)

Translated using Weblate (Czech)

Currently translated at 94.1% (16 of 17 strings)

Translated using Weblate (Czech)

Currently translated at 94.8% (37 of 39 strings)

Translated using Weblate (Czech)

Currently translated at 99.1% (481 of 485 strings)

Translated using Weblate (Finnish)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Czech)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (39 of 39 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Added translation using Weblate (Estonian)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (German)

Currently translated at 100.0% (39 of 39 strings)

Co-authored-by: Antonín JUDYTKA <judytka@judytka.cz>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: Lukáš Granatier <lukas.granatier@compacer.cz>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Ricky Tigg <ricky.tigg@gmail.com>
Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: on the road simson <d-youtube@kopaszhegy.hu>
Co-authored-by: polarwood <polarwood@users.noreply.hosted.weblate.org>
Co-authored-by: Максим Горпиніч <maksimgorpinic2005a@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/export/
Translate-URL: https://hosted.weblate.org/projects/kimai/flashmessages/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/
Translate-URL: https://hosted.weblate.org/projects/kimai/kiosk/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/kiosk/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/reporting/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Export
Translation: Kimai - Time tracking/Invoice-Templates
Translation: Kimai - Time tracking/Kiosk
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Reporting
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/User feedback (Flash messages and Alerts)
Translation: Kimai - Time tracking/Validations
Translation: Kimai - Time tracking/Wizard
